tests/PHPCR/Tests/Util/QOM/Sql2ScannerTest.php

```diff
@@ -124,6 +124,60 @@ public function testSQLEscapedStrings2()
         $this->expectTokensFromScanner($scanner, $expected);
     }
 
+    public function testSquareBrackets()
+    {
+        $sql = 'WHERE ISSAMENODE(file, ["/home node"])';
+
+        $scanner = new Sql2Scanner($sql);
+        $expected = [
+            'WHERE',
+            'ISSAMENODE',
+            '(',
+            'file',
+            ',',
+            '[',
+            '"/home node"',
+            ']',
+            ')',
+        ];
+
+        $this->expectTokensFromScanner($scanner, $expected);
+    }
+
+    public function testSquareBracketsWithoutQuotes()
+    {
+        $sql = 'WHERE ISSAMENODE(file, [/home node])';
+
+        $scanner = new Sql2Scanner($sql);
+        $expected = [
+            'WHERE',
+            'ISSAMENODE',
+            '(',
+            'file',
+            ',',
+            '[/home',
```

Comment:
This is probably the problem why the test in the other repository breaks. Before this was expected to be one token and now the space splitting is seperating it again.